### ISSUE TITLE
suport lcms2 color mgr? 

### ISSUE THREAD

**Issue by 947942434** [DanBloomberg/leptonica#529]
suport lcms2 color mgr? rgb to lab to cmyk? jpeg dpi is error?  tif  deep 32bit is ok ?

**Comment by DanBloomberg** [comment]
No, this is beyond the scope of color processing that is appropriate for Leptonica. It would also introduce another external library dependency into leptonica, something that has considerable downsides for most users and for support.

**Comment by amitdo** [comment]
@stweil,

Please close this old issue.
